templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -302,6 +302,28 @@
 			</div>
 		</div>
 
+		{{if and (or (.IsRepoAdmin) (.IsIssuePoster)) (not .Issue.IsPull)}}
+		<div class="ui divider"></div>
+		<div class="ui">
+			<span class="text"><strong>{{.i18n.Tr "repo.issues.confidential"}}</strong></span>
+			<div class="mt-3">
+				<form method="POST" action="{{.Issue.Link}}/private">
+					<input type="hidden" name="is_confidential" value="{{if .Issue.IsPrivate}}false{{else}}true{{end}}" />
+					{{$.CsrfTokenHtml}}
+					<label class="tooltip" data-content="{{if .Issue.IsPrivate}}{{.i18n.Tr "repo.issues.confidential.off_popup"}}{{else}}{{.i18n.Tr "repo.issues.confidential.on_popup"}}{{end}}">
+					<button data-tooltip="hi" class="fluid ui button df jc">
```

Comment:
hi? :D

Reply:
Woooppss that was me looking how to get tooltip working